game/server/voxel_types.lua

```diff
@@ -71,31 +71,31 @@ game.data.addVoxel {
     },
 }
 
-----  Wood Voxel Data
---game.data.addVoxel {
---    name = "openbuilder_wood",
---    description = "Wood",
---    collidable = false,
---    render = {
---        top = "logtop",
---        sides = "log",
---        bottom = "logtop",
---    },
---}
---
-----  Leaf Voxel Data
---game.data.addVoxel {
---    name = "openbuilder_leaf",
---    description = "Leaf",
---    type = game.VoxelType.Flora,
---    collidable = false,
---    render = {
---        top = "leaves",
---        sides = "leaves",
---        bottom = "leaves",
---    },
---}
---
+--  Wood Voxel Data
+game.data.addVoxel {
+    name = "openbuilder_wood",
+    description = "Common Wood",
+    collidable = false,
```

Comment:
Shouldn't you be able to collide with wood? Unsure about this since other solid blocks are also non-collidable.